components/dashboard/src/index.css

```diff
@@ -20,6 +20,20 @@
     p {
         @apply text-sm text-gray-400;
     }
+
+    .a-ul {
+        @apply underline;
+        text-decoration-thickness: 2px;
+        text-underline-offset: 3px;
+    }
+
+    .btn-login {
+        @apply cursor-pointer rounded-none border-none bg-gray-100 hover:bg-gray-200 text-sm text-gray-500;
```

Comment:
**issue**: Overall, we're going to need two radius corners, `6px` for all form elements like buttons, text fields, etc and `12px` for modals, list items, etc. We could [customize border radiuses](https://tailwindcss.com/docs/border-radius#customizing) but I think `rounded-md` is exactly what we need here, right?

```suggestion
        @apply cursor-pointer rounded-md border-none bg-gray-100 hover:bg-gray-200 text-sm text-gray-500;
```